.github/workflows/pre-merge-ci.yml

```diff
@@ -7,4 +7,7 @@ on:
 jobs:
   trigger-integration-tests:
     if: contains(github.event.pull_request.labels.*.name, 'run_integration_test')
-    uses: aqueducthq/aqueduct/.github/workflows/integration-tests.yml@main
\ No newline at end of file
+    uses: aqueducthq/aqueduct/.github/workflows/integration-tests.yml@test_premerge
```

Comment:
Once the premerge tests pass, I will return this back to @main and skip before merging.